cmd/keycloakb/keycloak_bridge.go

```diff
@@ -836,6 +865,59 @@ func config(logger log.Logger) *viper.Viper {
 	return v
 }
 
+func checkMigrationVersion(logger log.Logger, dbParams dbConfig, minMigrationVersion string) error {
+
+	migrationVersionDBConn, err := dbParams.openDatabase()
+	if err != nil {
+		logger.Log("msg", "could not create DB connection in order to check the migration version", "DB", dbParams.Database, "error", err)
+		return err
+	}
+	defer migrationVersionDBConn.Close()
+
+	var flywayVersion string
+	row := migrationVersionDBConn.QueryRow(`SELECT version FROM flyway_schema_history ORDER BY installed_rank DESC LIMIT 1;`)
+	err = row.Scan(&flywayVersion)
+	if err != nil {
+		logger.Log("msg", "could not query the flyway_schema_history", "error", err)
+		return err
+	}
+
+	// compare the two versions of the type x.y (major.minor)
+	minMajorRequired := strings.Split(minMigrationVersion, ".")
+	minMajorFlyway := strings.Split(flywayVersion, ".")
+
+	majorRequired, err := strconv.Atoi(minMajorRequired[0])
+	if err != nil {
+		logger.Log("msg", "could not convert string to int", "error", err)
```

Comment:
Could we 4 different messages for "could not convert string to int"... for example "could not convert majorRequired from string to int" ?